filebeat/docs/include/timezone-support.asciidoc

```diff
@@ -0,0 +1,17 @@
+[float]
+==== Timezone support
+
+This module parses logs that don't contain timezone information. For these logs,
+Filebeat reads the local timezone and uses it when parsing to convert the
+timestamp to UTC. The timezone to be used for parsing is included in the event
+in the `event.timezone` field.
+
+To disable this conversion, the `event.timezone` field can be removed with
+the `drop_fields` processor.
+
+If logs are originated from systems or applications with a different timezone to
```

Comment:
I assume that is for the syslog case etc?

Reply:
For example. But this can really happen in any case where the application generating the logs is configured with a different timezone than metricbeat.